modules/cnv-creating-vm.adoc

```diff
@@ -0,0 +1,60 @@
+// Module included in the following assemblies:
+//
+// * cnv_users_guide/cnv-create-vms.adoc
+
+[id="cnv-creating-vm_{context}"]
+= Creating a virtual machine from the CLI
+
+The `spec` object of the VirtualMachine configuration file references
+the virtual machine settings, such as the number of cores and the amount
+of memory, the disk type, and the volumes to use.
+
+Attach the virtual machine disk to the virtual machine by referencing
+the relevant PVC `claimName` as a volume.
+
+[NOTE]
+====
+https://kubernetes.io/docs/concepts/workloads/controllers/replicaset/[ReplicaSet]
+is not currently supported in {ProductName}.
+====
+
+See the Reference section for information about volume types and
+sample configuration files.
+
+.Domain settings
+|===
+|Setting | Description
+
+|cores
+|The number of cores inside the virtual machine. Must be a value greater than or equal to 1.
+
+|memory
+| The amount of RAM allocated to the virtual machine by the node. Specify the denomination with *M* for Megabyte or *Gi* for Gigabyte.
+
+|disks: name
+|The Name of the volume which is referenced. Must match the name of a volume.
```

Comment:
Check the capitalization here.
s/which/that

Reply:
Lower-cased name; using 'that'.

"The name of the volume that is referenced. Must match the name of a volume."

I'm not sure I would keep the second sentence. It seems redundant.